go/vt/sqlparser/ast.go

```diff
@@ -421,7 +421,7 @@ type SelectStatement interface {
 	SetLock(string)
 	SetOrderBy(OrderBy)
 	SetInto(*Into) error
-	HasIntoDefined() bool
+	HasIntoDefined() (*Into, bool)
```

Comment:
Feels a little non-standard to have a `Has` method that returns a field and a boolean value – from that prefix, I'd expect it to return a boolean value. How about renaming the method to `GetInto` and just having it return `*Into`? That would be a more common pattern and it would still be easy to test if the result == nil in the places where you were using the boolean.

Reply:
You're right! It makes sense to have GetInto() and use the result to check whether it's defined.